bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -392,73 +375,6 @@ public void setContext(@Nullable String context) {
         config.setContext(context);
     }
 
-    /**
-     * @deprecated use {@link Configuration#setBuildUuid(String)}
-     */
-    @Deprecated
-    @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
-    public void setBuildUUID(@Nullable final String buildUuid) {
-        config.setBuildUUID(buildUuid);
-    }
-
-    /**
-     * @deprecated use {@link Configuration#setFilters(String[])}
-     */
-    @Deprecated
-    public void setFilters(@Nullable String... filters) {
-        config.setFilters(filters);
-    }
-
-    /**
-     * @deprecated use {@link Configuration#setIgnoreClasses(String[])}
-     */
-    @Deprecated
-    public void setIgnoreClasses(@Nullable String... ignoreClasses) {
-        config.setIgnoreClasses(ignoreClasses);
-    }
-
-    /**
-     * @deprecated use {@link Configuration#setNotifyReleaseStages(String[])}
-     */
-    @Deprecated
-    public void setNotifyReleaseStages(@Nullable String... notifyReleaseStages) {
-        config.setNotifyReleaseStages(notifyReleaseStages);
-    }
-
-    /**
-     * @deprecated use {@link Configuration#setReleaseStage(String)}
-     */
-    @Deprecated
-    public void setReleaseStage(@Nullable String releaseStage) {
-        config.setReleaseStage(releaseStage);
-    }
-
-    /**
-     * @deprecated use {@link Configuration#setSendThreads(boolean)}
-     */
-    @Deprecated
-    public void setSendThreads(boolean sendThreads) {
-        config.setSendThreads(sendThreads);
-    }
-
-    /**
-     * @deprecated use {@link Configuration#setAutoCaptureSessions(boolean)}
-     */
-    @Deprecated
-    public void setAutoCaptureSessions(boolean autoCapture) {
-        config.setAutoCaptureSessions(autoCapture);
-
-        if (autoCapture) { // track any existing sessions
-            sessionTracker.onAutoCaptureEnabled();
-        }
-    }
-
-    private void setUserInternal(User user) {
```

Comment:
Why was this removed? Was it not used anywhere? Is user still observable if you set it via `setUser()`?

Reply:
This was removed unintentionally and has been added back in.